.github/workflows/tests.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         id: full-python-version
         shell: bash
         run: |
-          echo ::set-output name=version::$(python -c "import sys; print('-'.join(str(v) for v in sys.version_info))")
```

Comment:
Can you change this to the new `GITHUB_OUTPUT` syntax?

Reply:
Do you mean: `echo "{name}={value}" >> "$GITHUB_OUTPUT"`?

Reply:
Right